qi-doc/scribblings/interface.scrbl

```diff
@@ -173,14 +41,14 @@ See @secref["Flowing_with_the_Flow"] for ways to enter the @racket[☯] symbol i
   ]
 }
 
-@defform[(on (arg ...) flow-expr)]{
+@defform[(on (arg ...) @#,seclink["Syntax"]{@racket[floe]})]{
```

Comment:
This might be able to be extracted to make it easier to write:

```
@(define floe-link @seclink["Syntax"]{@racket[floe]})
…
@defform[(on (arg ...) #,floe-link) …
```

Maybe?

Reply:
Looks like this got converted to `tech` later on anyway, which simplifies things enough that maybe it's not worth it.